content/docs/deployment/csmoperator/modules/resiliency.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Resiliency
+linkTitle: "Resiliency"
+description: >
+  Pre-requisite for Installing Resiliency via Dell CSM Operator
+---
+
+The CSM Resiliency module for supported Dell CSI Drivers can be installed via the Dell CSM Operator. Dell CSM Operator will deploy CSM Resiliency sidecar.
+
+## Prerequisite
+
+The first thing to remember about _CSM for Resiliency_ is that it only takes action on pods configured with the designated label. Both the key and the value have to match what is in the resiliency module configuration. CSM for Resiliency emits a log message at startup with the label key and value it is using to monitor pods:
```

Comment:
rephrase this sentence

Reply:
How about this one?

![image](https://user-images.githubusercontent.com/109620911/235193584-6aad45fa-57a9-401d-858f-24d6e8fc518f.png)

Reply:
Fixed it as suggested.